lineman/app/models/proposal_model.coffee

```diff
@@ -88,3 +88,19 @@ angular.module('loomioApp').factory 'ProposalModel', (BaseModel, AppConfig) ->
         _.difference(@group().members(), @voters())
       else
         @recordStore.users.find(_.pluck(@didNotVotes(), 'userId'))
+
+    createOutcome: =>
+      @remote.postMember @id, "create_outcome",
```

Comment:
Hey I feel like I'd like this to read

```
@remote.postMember @id, 'create_outcome', @serialize('outcome') # or @serialize(['outcome'])
```

and let serialize take a list of fields to serialize (defaulting to everything). Thoughts?